MERP_Character_Sheet_BE/Controllers/GameClassController.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using MERP_Character_Sheet_BE.Models;
+using Newtonsoft.Json;

```

Comment:
I think we can remove this, it was introduced by me in one of my tests! but we can sort the imports later on

Reply:
Since the code is compiled. Does this affect the bundle size in the end? 
I mean, for Clean coding it's better to remove. but does it affect performance?

Reply:
I dont think so. I'm not 100% sure

Reply:
Ok, merging this one, I'll remove them on the next PR